staging/src/k8s.io/apiserver/pkg/cel/environment/base.go

```diff
@@ -80,6 +80,15 @@ var baseOpts = []VersionedOptions{
 			cel.CrossTypeNumericComparisons(true),
 			cel.OptionalTypes(),
 			library.Quantity(),
+
+			// cel-go v0.17.7 introduced CostEstimatorOptions.
+			// Previous the presence has a cost of 0 but cel fixed it to 1. We still set to 0 here to avoid breaking changes.
+			cel.CostEstimatorOptions(checker.PresenceTestHasCost(false)),
```

Comment:
Instead of putting this under  "IntroducedVersion" of 1.28, can we put it under the 1.0 group of settings at the top?  The behavior will be the same but I think that more clearly communicates that this isn't something we added in 1.28, it's something we added to preserve the behavior of CEL when we picked up v0.17.7.

Reply:
Makes sense. Updated :)